flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/validate/FlinkSqlConformance.java

```diff
@@ -168,8 +168,8 @@ public SqlLibrary semantics() {
     }
 
     @Override
-    public boolean allowCoercionStringToArray() {
-        return SqlConformanceEnum.DEFAULT.allowCoercionStringToArray();
+    public boolean allowLenientCoercion() {
```

Comment:
Suddenly was just renamed in Calcite https://github.com/apache/calcite/commit/b0420947ddeb2c9e18afe896dbf7adaf49a715a2